org.eclipse.xtext.tests/testdata/wizard-expectations/mavenTychoJ11/mavenTychoJ11.parent/mavenTychoJ11/pom.xml

```diff
@@ -36,26 +36,6 @@
 					<cleanupDaemonThreads>false</cleanupDaemonThreads><!-- see https://bugs.eclipse.org/bugs/show_bug.cgi?id=475098#c3 -->
```

Comment:
shouldn't we also remove

```xml
<includePluginDependencies>true</includePluginDependencies>
```

?

Reply:
if i had any clue why this is set to true in the first place

Reply:
I think it was originally put there to simplify the workflow (scanClasspath=true) which is no longer the default (since a very long time).

Reply:
Maybe I confused your idea, but I didn't mean `<cleanupDaemonThreads>false</cleanupDaemonThreads>`, I actually meant

```xml
<includePluginDependencies>true</includePluginDependencies>
```

That is, https://www.mojohaus.org/exec-maven-plugin/java-mojo.html#includePluginDependencies

I think it was added because the additional.bundles was not not taken into consideration, and, from what I understand, we don't want things like xtext.generator in the required bundles, since they are used only for running MWE2. So xtext.generator was added to additional.bundles. However, Tycho did not consider that (due to a bug, now fixed), and so we put them in the dependencies of the plugin (but then we had to tell maven exec to also consider plugin dependencies or MWE2 would not find the classes of xtext.generator and friends). If you try to remove that without Tycho 2.4.0 you get failure when running maven exec (it cannot find classes). With Tycho 2.4.0 that's not needed anymore, and I guess it avoids mixing classes from different classpath...

Reply:
i tried and removing does not work

Reply:
sorry I lost the workflow of the conversation ;)

removing what? and what does not work?

Reply:
the pluginDependencies thing.
i also lost overview.
thus i want to solve these stuff separately

Reply:
I think in the end it's really your call.
If you want to keep `<includePluginDependencies>true</includePluginDependencies>` then you can keep also the Maven dep on xtext-antlr... for the time being or forever ;)

Reply:
yes but this still has the commons io problem